float64 front_left_distance
float64 front_right_distance
float64 left_front_distance
float64 left_back_distance
float64 right_front_distance
float64 right_back_distance


